components/DepositTokensButton.tsx

```diff
@@ -30,12 +30,11 @@ export const DepositTokensButton = ({
   const depositTooltipContent = !connected
     ? 'Connect your wallet to deposit'
     : !hasTokensInWallet
-    ? "You don't have any governance tokens in your wallet to deposit."
-    : undefined
+      ? "You don't have any governance tokens in your wallet to deposit."
```

Comment:
it seems like you're using a different code formatter, any idea what's going on? FWIW I prefer this way of formatting ternaries

Reply:
using code spaces

Reply:
as in, using the codespace associated with the repo? interesting. how did you find that btw? I'm new to codespaces and just set it up for personal use.